e2e/tests/components/marquee-wizard/marquee-wizard.e2e-spec.ts

```diff
@@ -85,6 +85,40 @@ describe('Marquee Wizard Tests', () => {
         expect(attr).toBeNull();
     });
 
+    it('should disable the next button when the step is invalid and disableNextWhenInvalid = true', async () => {
+        // enable validation
+        await page.stepInvalidButton.click();
+        await page.disableNextWhenInvalidButton.click();
+
+        let next: ElementFinder = await page.getNextButton();
+        let attr = await next.getAttribute('disabled');
+        expect(attr).not.toBeNull();
+
+        // enable validation
```

Comment:
Also misleading, you are setting the step to be valid by clicking it a second time.